client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java

```diff
@@ -16,15 +16,14 @@
 
 package com.arcbees.beeshop.client.application;
```

Comment:
it's beestore, however, all the packages use that name. We planned to rename it, but it's still not done. It'll have to be done in a separate PR though.